### PR TITLE
Added missing tests

### PR DESCRIPTION
**Pull request by Unicus-Tom**
Now almost everything has a test for it.

### CODE REVIEW

File: RestApi/Sepes.RestApi/Model/PodInput.cs [hunk 1]
```diff
@@ -3,10 +3,17 @@ namespace Sepes.RestApi.Model
 
     public class PodInput
     {
-        public string podName { get; set; }
-        public int studyID { get; set; }
-        public string podTag { get; set; }
-        public int podID { get; set; }
+        public int id { get; set; }
```
Comment (Kris2ffer) [inline RestApi/Sepes.RestApi/Model/PodInput.cs]: These changes are out of scope.
It's likely that frontend also must be changed based on this because it must use the same names.

Comment (Unicus-Tom) [reply RestApi/Sepes.RestApi/Model/PodInput.cs]: I will revert these changes. Its true they where out of scope

**Review by Kris2ffer — APPROVED** [review]

**Review by gjersvik — APPROVED** [review]

**Review by Kris2ffer — APPROVED** [review]
